### ISSUE TITLE
Big asset files cause lagging (Object explorer tree)

### ISSUE THREAD

**Issue by jyy1082** [tentone/nunuStudio#142]
Description

When I imported a font (13.81MB) into asset explorer, even I did not use it, the running program goes always lagging.

 Run this .nsp file can see this issue: https://github.com/jyy1082/test/blob/master/https___nunustudio.org_learn_basics_02_script_include%E2%80%94%E2%80%941.nsp?raw=true

Version
 - <0.9.3> 

Platform
 - Web Version [X]

**Comment by tentone** [comment]
Hi

The problem here is not the font file actually, that example gets slow due to the 3D Text changes, when a 3D text is changed the hole geometry gets recalculated.

I will add a new issue to try and add proper 2D text with distance fields that should be really fast when changing text.

The font has no actual impact on performance, it makes the load time bigger but its just that.

Thanks

**Comment by jyy1082** [comment]
Thanks for your response.

I also tried to import a large .obj file (59.8 MB ) into asset explorer.  After this file loaded, the editor goes really slow. For example, if I add one script and open it, I have to wait for a while for my every operation.  I also tried threejs.org/editor using this file, but there is no this kind of issue.

This is .obj file which I used:
https://drive.google.com/open?id=1arFBoOz2YKjUFvAfUT5VTwUyVm-qBsID
